src/makeCalculator.js

```diff
@@ -4,7 +4,39 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  return {
+    result: 0,
+
+    add(value) {
+      this.result += value;
+    },
+
+    subtract(value) {
+      this.result -= value;
+    },
+
+    multiply(value) {
+      this.result *= value;
+    },
+
+    divide(value) {
+      if (value !== 0) {
+        this.result /= value;
+      }
```

Comment:
[CODE STYLE] - Remember to use the arithmetic operator with the assignment operator `=`. Instead of `this.result /= value;` you can write `this.result /= value;` directly, as the division assignment operator already includes the assignment operation. (Checklist item 2)